docs/STRATEGY_SELECTION.md

```
@@ -0,0 +1,452 @@
+| that-site.com/ | firecrawl | Native returns 403 |
+```
+
+### Want to force native for API endpoints
+
+**Solution:** Add to strategy config:
+
+```markdown
+| api.site.com/ | native | Simple JSON API |
+```
+
+### Not sure which strategy works best
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)